### PR TITLE
36 setup sequelize models

### LINKED ISSUE #36: Set up Sequelize Models
- [ ] Set up initial sequelize models
**Acceptance Criteria**
- [x] Ensure that Models can be used to access the database.

Note: This issue is being used as a prerequisite for issues 28, 29, and 30, since all of these will use Sequelize.

Estimated time: 30 minutes.

### PR DESCRIPTION
**Pull request by MongoSP**
**Addressed Issue(s)**

closes #36 - Set Up Sequelize Models

## Acceptance Criteria

- [x] Sequelize Models are correct.
- [x] SQL Query file for creating Tables has been properly updated to be compatible with Sequelize.
- [x] All models are exported from this file.

### Special notes for reviewer
This file is meant to export the models for Sequelize. The connection in sequelize is replacing our original means of connection. This custom module will still need to be updated so as to use the Heroku database and/or the database of other users.

### CODE REVIEW

**Comment by DRSisco** [comment]
@MongoSP to reference an issue or PR you need to put the number with the `#` 
>closes # 36 - Set Up Sequelize Models
Will not link to the issue nor with it trigger
https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

I suggest you change it to 
>closes #36 - Set Up Sequelize Models

**Comment by MongoSP** [comment]
Thank you for your quick reply, I have updated the description.

File: sequelizeDatabase/modelSetup.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('localDatabaseSean','root','ConnectedNetwork1',{
```
Comment (DRSisco) [inline sequelizeDatabase/modelSetup.js]: `localDatabaseSean` 🤣 
I like it. We should probably make an issue to extract this to a `.env` file

File: sequelizeDatabase/modelSetup.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+sequelize.authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+  const Individuals = sequelize.define('individuals',{
```
Comment (DRSisco) [inline sequelizeDatabase/modelSetup.js]: From here on you added extra indentation. Happens, might be able to look for something in your IDE to notify you of that.

Comment (DRSisco) [inline sequelizeDatabase/modelSetup.js]: I think you might find it useful to separate your models into separate files.

**Pull request merged**